board/arch.md

```diff
@@ -0,0 +1,115 @@
+# mlpack on RPI and resource constrained devices
+
+Here is the list of possible architectures that I have in mind,
+this list is subject to modifications, new architectures will be added,
+and some of these will be deleted.
+
+(This table should not render in Markdown)
+
+|  Possible architecture  | Possible toolchains | Known Devices      |
+|---------------------------------------------------------------------
+|     Alpha               |       GCC           |      ?
+----------------------------------------------------------------------
+| **ARMv6** (32 bit)      |       GCC           | RPI 0, RPI 1
+| **ARMv7** (32 bit)      |       GCC           | RPI 2, Beagleboard, Cortex-M devices
+| **ARMv8** (64 bit)      |       GCC           | RPI 2(1.2), RPI 3, RPI 4, Banan Pi,
+|
+| **AVR**                 |       GCC            |  Arduino
+------------------------------------------------------------------
+| Blackfin                |       GCC            |   ?
+-----------------------------------------------------------------------
+| Epiphany                |      GCC 4.8         | Parallella
```

Comment:
If I remember right they discontinued the Parallella project and Epiphany V will probably never see the light, nevertheless I have two boards that are just waiting to crunch some numbers.

Reply:
@zoq That is good to hear, I had doubts about this board. It is perfect to know that we have the hardware ready for testing